pkg/services/alerting/conditions/evaluator.go

```diff
@@ -10,8 +10,8 @@ import (
 )
 
 var (
-	defaultTypes = []string{"gt", "lt"}
-	rangedTypes  = []string{"within_range", "outside_range"}
+	defaultTypes = []string{"gt", "lt", "eq", "ne", "ge", "le"}
```

Comment:
This file is for the deprecated (planning to be removed) dashboard alerts. You can remove these changes as we're no longer accepting changes for it. The rest of the changes for Grafana Alerting are fine!